### DISCUSSION TITLE
Urql Devtools alternative for debugging

### DISCUSSION THREAD

**Discussion by lsarni** [urql-graphql/urql#3625] [Q&A]
Since the [Chrome extension isn't available anymore](https://github.com/urql-graphql/urql-devtools/issues/408), and the [Firefox one appears to be broken](https://github.com/urql-graphql/urql-devtools/issues/399). Are there other ways of knowing what is being cached? I'm trying to add Graphcache to my app, but having a hard time with it.

**Comment by Khuder-Karim** [comment]
I encountered the same issue. I resolved it by building the extension locally and adding it to Chrome in developer mode using 'Load unpacked.

1) Clone repository https://github.com/urql-graphql/urql-devtools
2) pnpm build:extension. Using node 18
3) Load dist/extensio on chrome://extensions/ (https://webkul.com/blog/how-to-install-the-unpacked-extension-in-chrome/)

**Comment by itpropro** [comment]
Any updates from the urql team @jtomaszewski @kitten what the way forward is for debugging? Not having dev tools, especially when dealing with cache related problems can be very challenging. The repo also hasn't been updated for 2 years and the extensions were last published 4 years ago.
